h2o-core/src/test/java/water/parser/ChunksizeTest.java

```diff
@@ -79,7 +79,7 @@ public void run() throws IOException {
                   fail = true;
                 }
 
-                if (chunkSize >= Value.MAX) {
+                if (chunkSize >= (1<<28)) { //256MB
```

Comment:
We should have the parser-private constant for that - since it is referenced from several places.